### PR TITLE
Docsp 16491 typos

### PR DESCRIPTION
**Pull request by biniona-mongodb**
## Pull Request Info

Fix Typos in Index, Quick Start, and Usage Examples

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-16491

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/778514e/java/docsworker-xlarge/DOCSP-16491-Typos/quick-start/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/includes/quick-start/atlas-setup.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Set up a Free Tier Cluster in Atlas
 After setting up your Java project dependencies, create a MongoDB cluster
 where you can store and manage your data. Complete the
 :atlas:`Get Started with Atlas </getting-started>` guide to set up a new
-Atlas account, free tier MongoDB cluster, load datasets, and
+Atlas account, build a free tier MongoDB cluster, load datasets, and
```
Comment (ccho-mongodb) [inline source/includes/quick-start/atlas-setup.rst:7]: I think "create and launch" are more appropriate. The user doesn't perform any builds.

**Review by ccho-mongodb — COMMENTED** [review]
Before I continue the review, I'm not sure many of these changes fit the scope of "fix typos", so I'd like to get some clarification in the issue. 

Some of the changes are typo fixes.

Some of them are style guide fixes which is great, but does not fix all of them such as "Use first-person plural pronouns (we, our) judiciously."

Some of them are arbitrary word choice changes that do not add any value.

Some changes introduce new grammatical issues.

Let's avoid this type of open-ended PR if possible as they often increase significantly in scope and narrow it down to the specific items it addresses.

File: source/includes/quick-start/atlas-setup.rst (hunk 1 above)
Comment (biniona-mongodb) [reply source/includes/quick-start/atlas-setup.rst]: I agree "create and launch" is more appropriate here.

**Comment by biniona-mongodb** [comment]
Thanks for the note Chris. We discussed this synchronously in our weekly check-in and I better understand how an open ended PR can be harmful to progressing on a task. I went through and removed what I thought were opinionated changes in this PR. I left in style-guide fixes as it felt wasteful to loose this work, but decided to not implement style guide fixes beyond capitalizing list items in future typo fix PRs.

**Review by kyuan-mongodb — APPROVED** [review]
Great work on catching all these grammatical errors! LGTM with a few nits/suggestions.

File: source/usage-examples/insertOne.txt [hunk 2]
```diff
@@ -32,7 +32,7 @@ collection.
    :language: java
 
 When you run the example, you should see output that resembles the following
-with the inserted document ``ObjectId`` in the value field:
+with the inserted document's ``ObjectId`` in the value field:
```
[the remark this answers is not in the document]
Comment (biniona-mongodb) [reply source/usage-examples/insertOne.txt:35]: I think "'' indicates that it is possessive rather plural (the `ObjectId` possessed by the document). Do you think the possessive form is incorrect here?

File: source/includes/quick-start/atlas-setup.rst [hunk 3]
```diff
@@ -33,8 +33,9 @@ want to connect to as shown below.
    :alt: Atlas Connection GUI cluster selection screen
 
 Proceed to the **Connect Your Application** step and select the Java driver.
-Then, select the "Connection String Only" tab and click the **Copy**
-button to copy the *connection string* to your clipboard as shown below.
+Enter "4.1 or Later" for the version.
```
[the remark this answers is not in the document]
Comment (biniona-mongodb) [reply source/includes/quick-start/atlas-setup.rst:36]: Good catch, I agree "select" is a better word here.

File: source/usage-examples/insertMany.txt [hunk 4]
```diff
@@ -32,7 +32,7 @@ collection.
    :language: java
 
 When you run the example, you should see output that resembles the following
-with the inserted document ``ObjectId`` in each of the value fields:
+with the inserted documents' ``ObjectId`` values in each of the value fields:
```
[the remark this answers is not in the document]
Comment (biniona-mongodb) [reply source/usage-examples/insertMany.txt:35]: I think "'' indicates that it is possessive rather plural (the `ObjectId` possessed by the documents). Do you think the possessive form is incorrect here?

File: source/usage-examples/insertOne.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (kyuan-mongodb) [reply source/usage-examples/insertOne.txt:35]: ah I see, you're right, I misread it initially

**Pull request merged**